### ISSUE TITLE
Can't verify accuracy with pre-trained Resnet50 model.

### ISSUE THREAD

**Issue by ddokupil** [PaddlePaddle/models#1223]
Cannot obtain any meaningful results when when trying to reproduce ResNet50 accuracy (76.63%/93.10%).

Using Image Net val dataset:
val/659/ILSVRC2012_val_00046108.JPEG 659
val/659/ILSVRC2012_val_00044030.JPEG 659
val/659/ILSVRC2012_val_00042898.JPEG 659
val/659/ILSVRC2012_val_00041481.JPEG 659
val/659/ILSVRC2012_val_00039493.JPEG 659
val/659/ILSVRC2012_val_00037864.JPEG 659
val/659/ILSVRC2012_val_00037578.JPEG 659
val/659/ILSVRC2012_val_00033095.JPEG 659
val/659/ILSVRC2012_val_00031091.JPEG 659
val/659/ILSVRC2012_val_00030154.JPEG 659
val/659/ILSVRC2012_val_00029819.JPEG 659
val/659/ILSVRC2012_val_00029360.JPEG 659
val/659/ILSVRC2012_val_00028777.JPEG 659

And provided model I keep getting:

-----------  Configuration Arguments -----------
class_dim: 1000
image_shape: 3,224,224
model: ResNet50
pretrained_model: /data/resnet_50/115
use_gpu: 0
with_mem_opt: 1
------------------------------------------------
Test-0-score: [30.961004], class [550]
Test-1-score: [16.557055], class [550]
Test-2-score: [21.98035], class [550]
Test-3-score: [14.857814], class [505]
Test-4-score: [12.008922], class [550]
Test-5-score: [19.638456], class [550]
Test-6-score: [16.092129], class [813]
Test-7-score: [24.678347], class [550]
Test-8-score: [18.044142], class [550]
Test-9-score: [22.066696], class [550]
Test-10-score: [15.797653], class [550]
Test-11-score: [20.66053], class [550]
Test-12-score: [16.733633], class [482]
Test-13-score: [15.197915], class [550]
Test-14-score: [18.904078], class [550]
Test-15-score: [27.324083], class [550]
Test-16-score: [16.564373], class [550]
Test-17-score: [14.2177515], class [813]

And:

-----------  Configuration Arguments -----------
batch_size: 1
class_dim: 1000
image_shape: 3,224,224
model: ResNet50
pretrained_model: /data/resnet_50/115
use_gpu: 0
with_mem_opt: 1
------------------------------------------------
Testbatch 0,loss 12.8082323074, acc1 0.0,acc5 0.0,time 0.33 sec
Testbatch 10,loss 25.0121593475, acc1 0.0,acc5 0.0,time 0.16 sec
Testbatch 20,loss 20.8600234985, acc1 0.0,acc5 0.0,time 0.16 sec
Testbatch 30,loss 21.9461364746, acc1 0.0,acc5 0.0,time 0.16 sec
Testbatch 40,loss 14.2183275223, acc1 0.0,acc5 0.0,time 0.17 sec
Testbatch 50,loss 22.2879886627, acc1 0.0,acc5 0.0,time 0.19 sec
Testbatch 60,loss 15.6957626343, acc1 0.0,acc5 0.0,time 0.16 sec
Testbatch 70,loss 18.4210853577, acc1 0.0,acc5 0.0,time 0.16 sec
Testbatch 80,loss 26.5067806244, acc1 0.0,acc5 0.0,time 0.17 sec

or

-----------  Configuration Arguments -----------
batch_size: 32
class_dim: 1000
image_shape: 3,224,224
model: ResNet50
pretrained_model: /data/resnet_50/115
use_gpu: 0
with_mem_opt: 1
------------------------------------------------
Testbatch 0,loss 16.9630355835, acc1 0.03125,acc5 0.03125,time 4.76 sec
Testbatch 10,loss 16.6469154358, acc1 0.0,acc5 0.0,time 24.60 sec
Testbatch 20,loss 16.7011985779, acc1 0.0,acc5 0.0,time 3.54 sec
Testbatch 30,loss 13.4966154099, acc1 0.0,acc5 0.0,time 3.48 sec

**Comment by qingqing01** [comment]
The label index we used to trainned model is not same with yours:  

```
val/ILSVRC2012_val_00000001.JPEG 65
val/ILSVRC2012_val_00000002.JPEG 970
val/ILSVRC2012_val_00000003.JPEG 230
val/ILSVRC2012_val_00000004.JPEG 809
val/ILSVRC2012_val_00000005.JPEG 516
val/ILSVRC2012_val_00000006.JPEG 57
val/ILSVRC2012_val_00000007.JPEG 334
val/ILSVRC2012_val_00000008.JPEG 415
val/ILSVRC2012_val_00000009.JPEG 674
val/ILSVRC2012_val_00000010.JPEG 332
val/ILSVRC2012_val_00000011.JPEG 109
val/ILSVRC2012_val_00000012.JPEG 286
```

Please try to download label file list by:

```bash
wget http://paddle-imagenet-models.bj.bcebos.com/ImageNet_label.tgz
```


Our released model is trained on this label index.  See :  https://github.com/PaddlePaddle/models/blob/develop/fluid/image_classification/data/ILSVRC2012/download_imagenet2012.sh#L37

**Comment by qingqing01** [comment]
The picture `val/ILSVRC2012_val_00029819.JPEG` label index is 550 in our list, it is  an 'espresso maker' .

```
val/ILSVRC2012_val_00029819.JPEG 550
```

But your picture is labled  659.

So please update your label list.

**Comment by ddokupil** [comment]
It looks way better now, thanks.
Resolved.
